dces-drc-integration/src/integrationTest/java/uk/gov/justice/laa/crime/dces/integration/service/FdcTestDataCreatorServiceTest.java

```diff
@@ -292,7 +291,11 @@ public void checkNullRequest() throws InterruptedException {
      */
     private void checkRequestAndBody(String expectedMethod, String expectedRequestPath, String expectedRequestBody) throws InterruptedException {
         RecordedRequest recordedRequest = checkRequest(expectedMethod, expectedRequestPath);
-        assertEquals(expectedRequestBody, recordedRequest.getBody().readUtf8());
+        String actualRequestBody = recordedRequest.getBody().readUtf8();
+        log.info("Expected Request Body: " + expectedRequestBody);
```

Comment:
Minor question: Is it helpful to log this even when the expected and actual request bodies match?

Reply:
And if they're different, wouldn't `assertEquals` show the difference anyway?